internal/atlas/image.go

```diff
@@ -440,6 +440,27 @@ func (i *Image) DrawTriangles(srcs [graphics.ShaderSrcImageCount]*Image, vertice
 	i.drawTriangles(srcs, vertices, indices, blend, dstRegion, srcRegions, shader, uniforms, fillRule)
 }
 
+func DrawTrianglesMRT(dsts [graphics.ShaderDstImageCount]*Image, srcs [graphics.ShaderSrcImageCount]*Image, vertices []float32, indices []uint32, blend graphicsdriver.Blend, dstRegion image.Rectangle, srcRegions [graphics.ShaderSrcImageCount]image.Rectangle, shader *Shader, uniforms []uint32, fillRule graphicsdriver.FillRule) {
```

Comment:
Why not unifying DrawTriangles with this?

Reply:
I originally thought I might have a lot of additional things to write and wanted to make it distinct, but I can merge them now!
Another point is that I was afraid of the cost of the copy of an extra array of 8 pointers in all paths with (5-6x layer depth, ebiten,ui,buffered,mipmap,atlas, etc..) which would apply to all other standard two-triangles `DrawImage` call.

Reply:
> Another point is that I was afraid of the cost of the copy of an extra array of 8 pointers in all paths with (5-6x layer depth, ebiten,ui,buffered,mipmap,atlas, etc..) which would apply to all other standard two-triangles DrawImage call.

I don't think this would be problematic in terms of performance. Also, I'm not sure we really want 8 images (4 would be enough for example?).